godtools/App/Features/SetupParallel/SetupParallelLanguageView.swift

```diff
@@ -6,19 +6,19 @@
 //  Copyright © 2021 Cru. All rights reserved.
 //
 
-import Foundation
 import UIKit
 
-class SetupParallelLanguageView: UIViewController {
+class SetupParallelLanguageView: UIViewController, UIGestureRecognizerDelegate {
```

Comment:
I don't see any UIGestureRecognizerDelegate methods being used in this class. https://developer.apple.com/documentation/uikit/uigesturerecognizerdelegate

No need to add the delegate for UITapGestureRecognizer.